source/model/matchConfig.mc

```diff
@@ -7,11 +7,13 @@ class MatchConfig {
     private var superTie;
     private var numberOfSets;
     private var goldenPoint;
+    private var numberOfDeuces;
 
     function initialize() {
         superTie = false; 
         numberOfSets =  UNLIMITED_SETS;   
-        goldenPoint = true; 
+        goldenPoint = true;
+        //numberOfDeuces = 1;
```

Comment:
commented line?

Reply:
No need, at first I thought I needed to initialize the the variable, but then I found that how to reset the statistics